docs/wiki/update-alz-custom-policies-to-latest.md

```diff
@@ -0,0 +1,280 @@
+# Introduction
+
+This article describes how to update ALZ custom policies and policy initiatives to latest versions of ALZ custom policies. The guidance provided in this document describes manual steps for performing the update, based on a set of specific policies and initiatives, with PowerShell.
+
+> Note: If you are already managing Azure Policies through Infrastructure as Code (IaC), you may not want to be updating your Azure Policies outside of your pipeline. This article assume a manual approach to updating Azure landing zone (ALZ) policies.
+
+> Important: To carry out the instructions below, the operator will require Resource Policy Permissions at the root of the ALZ management group hierarchy.
+
+## Detect updates to policy
+
+1. To determine if there has been updates to ALZ your first reference should be [What's New](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new). Any updates to policies or other ALZ related artifacts will be reflected here upon release. An example of what that will look like can be seen [here](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new#policy).
+
+2. An alternatively or supplementary tool is [Azure Governance Visualizer](https://github.com/JulianHayward/Azure-MG-Sub-Governance-Reporting) which can be run in your environment and reveal information about the current state of policies and policy assignments. Part of the output of Azure Governance Visualizer is Azure Landing Zones (ALZ) Policy Version Checker which will allows you to see all **outDated** ALZ policies in your environment (see figure 1).
+![AzGovViz-ALZ-Policy-Checker](media/AzGovViz-ALZ-Policy-outDated.png)
+*Figure 1: Azure Governance Visualizer filtering on outDated ALZ policies*
+
+> Note that Azure Governance Visualizer requires permissions in your tenant as described [here](https://github.com/JulianHayward/Azure-MG-Sub-Governance-Reporting#permissions-overview)
+
+## Updating scenarios
+
+These are the following scenarios for ALZ custom policies being updated to latest versions of the custom ALZ policies, listed in increasing order of complexity:
+
+1. One or more ALZ custom policies, whether assigned or not at one or more scopes in your Azure estate, is outdated by a newer ALZ custom policy. The process for managing this is described in [Updating a one or more ALZ custom policy to newer ALZ custom policies](#updating-a-one-or-more-alz-custom-policies-to-newer-alz-custom-policy).
+
+2. One or more ALZ custom policies, assigned at one or more scopes in your Azure estate, is outdated by a newer ALZ custom policy with **updated parameters**. The process for managing this is described in [Updating a one or more ALZ custom policies to a newer ALZ custom policy with updated parameters](#updating-one-or-more-alz-custom-policies-to-newer-alz-custom-policy-with-updated-parameters)
```

Comment:
2. One or more ALZ custom policies, assigned at one or more scopes in your Azure estate, is superseded by a newer version of that same ALZ custom policy with **updated parameters**. The process for managing this is described in [Updating a one or more ALZ custom policies to a newer ALZ custom policy with updated parameters](#updating-one-or-more-alz-custom-policies-to-newer-alz-custom-policy-with-updated-parameters)
```suggestion
2. One or more ALZ custom policies, assigned at one or more scopes in your Azure estate, is outdated by a newer ALZ custom policy with **updated parameters**. The process for managing this is described in [Updating a one or more ALZ custom policies to a newer ALZ custom policy with updated parameters](#updating-one-or-more-alz-custom-policies-to-newer-alz-custom-policy-with-updated-parameters)
```